src/reynir_correct/main.py

```diff
@@ -141,6 +141,12 @@ class AnnResultDict(TypedDict):
     action="store_true",
 )
 
+parser.add_argument(
+    "skip_spelling_suggestion",
+    "--sss",
+    action="store_true",
+    help="Create an analysis report for token results",
```

Comment:
Not sure I understand this help text. Can it be made clearer?

Reply:
Done!